INSTALL.md

```diff
@@ -82,7 +95,7 @@ Other Fedora versions or distributions may require some adaptation.
    ```bash
    alias rhsm-service="sudo \
        PYTHONPATH=/path/to/subscription-manager/src \
-       $(which python3) \
+       $(which python3)  \
```

Comment:
Extra space

Reply:
Oh. I spotted it myself, but didn’t commit and push it. 🤦🏻‍♂️